com.unity.ml-agents/Runtime/MLAgentsDefines.cs

```diff
@@ -0,0 +1,3 @@
+#if MLA_UNITY_ANALYTICS_MODULE || !UNITY_2019_4_OR_NEWER
```

Comment:
Just to clarify, this will stay enabled in 2018.4? (Not relevant for long, but I think this is the right move).

Reply:
yes